logistics_project/apps/malawi/warehouse/runner.py

```diff
@@ -497,10 +497,15 @@ def update_consumption_times(since):
     """
     Update the consumption time_with_data values for any supply point / product pairings
     where the supply point doesn't explicitly supply the product.
+    This function only works against base level data.
     """
     # any consumption value that was touched potentially needs to have its
     # time_needing_data updated
-    consumptions_to_update = CalculatedConsumption.objects.filter(update_date__gte=since)
+    consumptions_to_update = CalculatedConsumption.objects.filter(
+        Q(product__type__base_level=BaseLevel.HSA, supply_point__type__code=SupplyPointCodes.HSA) |
+        Q(product__type__base_level=BaseLevel.FACILITY, supply_point__type__code=SupplyPointCodes.FACILITY),
+        update_date__gte=since
+    )
```

Comment:
yeah this only gets run during the warehouse runner, it won't be called by the management command introduced in this pr.

i think i misunderstood what this was doing - i thought it was pulling records updated since the end of the last warehouse run, but it's only pulling records updated since the beginning of the current run. so the problem i was envisioning won't be a problem and i reverted this commit.

it does seem that it would only ever pull base level data given the point at which this is run in the runner, but including that big or statement in there on such a big table would just slow down the query.